### PR TITLE
add on routing

### PR DESCRIPTION
**Pull request by eyeseast**
Adds hash URLs to `App.svelte`, based on this plan: https://www.notion.so/muckrock/Add-On-deep-linking-fe36e206f06146c4aa1a340f23d33ec3

Still to do: Pre-filling the add-on dispatch form based on query params, but that'll be next.

Try locally:

- https://www.dev.documentcloud.org/app#add-ons/MuckRock/Klaxon/1
- https://www.dev.documentcloud.org/app#add-ons/MuckRock/Klaxon
- https://www.dev.documentcloud.org/app#add-ons

### CODE REVIEW

File: src/common/dialog/AddonDispatchTsDialog.svelte [hunk 1]
```diff
@@ -1,41 +1,28 @@
-<script lang="ts">
-  import Button from "@/common/Button";
-  import AddonRun from "@/common/AddonRun";
+<script>
```
Comment (mitchelljkotler) [inline src/common/dialog/AddonDispatchTsDialog.svelte:1]: I could probably use some description as to why this file changed so much, it is a little hard to follow

**Review by mitchelljkotler — COMMENTED** [review]
Overall looks good.  It might be useful to separate out formatting/clean up changes from functional changes to make the diffs more useful, though I know that can be a pain

File: src/common/dialog/AddonDispatchTsDialog.svelte (hunk 1 above)
Comment (eyeseast) [reply src/common/dialog/AddonDispatchTsDialog.svelte:1]: This one had a bunch of little things that were touched, plus one big formatting change. The big one was an outer `<div>` that wasn't doing anything, so I deleted it. That caused an indentation change on the whole component. But it also had a couple unused functions and imports, which I deleted, and then I added stuff, so there's a lot. This file just does a bunch of things.
